bin/process.sh

```diff
@@ -6,10 +6,15 @@ shift $[$#-2]
 
 input=$1
 output=$2
-# target size in KB
-THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300}
+
+THUMBNAIL_SIZE=${THUMBNAIL_SIZE:-300} # target size in KB
 TILER_BASE_URL=${TILER_BASE_URL:-http://tiles.openaerialmap.org}
 
+# Although TMPDIR is usually set by the OS, it doesn't seem to be getting passed through
+# Docker and Node's `spawn()`, so no harm done in just defaulting to `/tmp` in such cases.
+# There may be clues here as the underlying issue: https://github.com/npm/npm/issues/4531
+TMPDIR={TMPDIR:-/tmp}
```

Comment:
I can't be certain, but the impression I had is that `/tmp` is generally manged by something called `tmpwatch` which applies whether `tmpfs` is used or not. Perhaps what you saw was that files were being downloaded but not persisted to the filesystem? Because `tmpfs` is some fraction the size of RAM.

Basically there needs to be a note in the README that looots of `/tmp` is needed whereever/however it is. Even using `$HOME/tmp` would need the host OS to have `tmpwatch` or somesuch because otherwise the disk is just going to fill up.